core/templates/dev/head/pages/about-page/about-page.controller.ts

```diff
@@ -12,91 +12,99 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-require('base_components/BaseContentDirective.ts');
-require(
-  'components/common-layout-directives/common-elements/' +
-  'background-banner.directive.ts');
-
```

Comment:
This should have been kept in this file. Same for the other pages.

Reply:
Done. The other pages dont have a controller file so they are in the scripts file.